services/install-grocy.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/grocy
+
+  # create yml(s)
+  cat << EOF > /srv/grocy/grocy.yml
+version: "2.1"
+services:
+  grocy:
+    image: linuxserver/grocy
+    environment:
+      - PUID=${PUID}
+      - PGID=${PUID}
+      - TZ=${TZ}
+      - PASSWORD=${PASSWORD}
+      - DB_TYPE=${DB_TYPE}
+      - DB_NAME=${DB_NAME}
+      - DB_HOSTNAME=${DB_HOSTNAME}
+      - DB_USERNAME=${DB_USERNAME}
+      - DB_PASSWORD=${DB_PASSWORD}
```

Comment:
The dollar signs need to be escaped so that they aren't expanded when creating the file (results in empty variables).
`- PUID=\${PUID}`